sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 - The library now targets the service's v2.0 API, instead of the v2.0-preview.1 API.
 - Updated version number from `1.0.0-preview.5` to `3.0.0-preview.5`.
+- Added models `RecognizeCustomFormsOptions`, `RecognizeReceiptsOptions`, and `RecognizeContentOptions` instead of a generic `RecognizeOptions` to support passing configurable options to recognize API's.
+- Added model`TrainingOptions` to support passing configurable options to training API's. This type now includes `TrainingFileFilter`.
```

Comment:
nit: missing blank space between `model` and `TrainingOptions`.

Reply:
nit: regarding the two additions to the changelog, I believe [we should drop the apostrophe when pluralizing acronyms](https://www.wikihow.com/Pluralize-Abbreviations,-Acronyms,-and-Initialisms). So: `APIs` instead of `API's`. I think a lot of people use the apostrophe anyway, so I wouldn't call this an issue.